Automattic/jetpack
# My Plan styling update

Fixes #

## Proposed changes:

* Edited mostly CSS styles for My Plan page
* Some changes to dops button and card CSS
* Expired plan is still not updated. That will be added in another PR

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to My Plan

It should look like this:

<img width="1207" alt="Screenshot 2023-04-24 at 15 24 42" src="https://user-images.githubusercontent.com/1242807/234009652-3b2f2615-8a16-4dda-bd18-1d15a5d11884.png">

* Try adding/removing products and plans.
* Try deactivating Akismet, Backup, Social
